File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/MT_POU_CHARGECAPTURE_HEADER.ts
export class MT_POU_CHARGECAPTURE_HEADER {
    public TRANSACTION_ID: number;
    public PATIENT_ID: string;
    public CAPTURE_DATE_TIME:  Date;
    public USER_ID: string;
    public ACCOUNT_ID: string;
    public EXAM_ID: string;
    public COMMENTS: string;
    public REVIEWED: boolean;
    public DEPARTMENT_ID: string;
    public WORKSTATION_ID: string;
    public MM_LAST_STATUS_DATE?: Date;
    public MM_STATUS: string;
    public CHARGE_LAST_STATUS_DATE?: Date;
    public CHARGE_STATUS: string;
    public REVIEW_USER_ID: string;
    public REVIEW_DATE?: Date;
    public PHYSICIAN_ID: string;
    public REASON_CODE: string;
    public OTHER_DEPTID: string;
    public BED_NO: string;
    public TRANSACTION_TYPE?: number;
    public CASE_ID: string;
    public PREF_LIST_ID: string;
    public APP_ID?: number;
    public CRASH_CART_ID: string;
}

File: RSPAClient/Core/AtPar.POCOEntities/MT_POU_CHARGECAPTURE_HEADER.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_POU_CHARGECAPTURE_HEADER
    {
        public long TRANSACTION_ID { get; set; }
        public string PATIENT_ID { get; set; }
        public System.DateTime CAPTURE_DATE_TIME { get; set; }
        public string USER_ID { get; set; }
        public string ACCOUNT_ID { get; set; }
        public string EXAM_ID { get; set; }
        public string COMMENTS { get; set; }
        public bool REVIEWED { get; set; }
        public string DEPARTMENT_ID { get; set; }
        public string WORKSTATION_ID { get; set; }
        public Nullable<System.DateTime> MM_LAST_STATUS_DATE { get; set; }
        public string MM_STATUS { get; set; }
        public Nullable<System.DateTime> CHARGE_LAST_STATUS_DATE { get; set; }
        public string CHARGE_STATUS { get; set; }
        public string REVIEW_USER_ID { get; set; }
        public Nullable<System.DateTime> REVIEW_DATE { get; set; }
        public string PHYSICIAN_ID { get; set; }
        public string REASON_CODE { get; set; }
        public string OTHER_DEPTID { get; set; }
        public string BED_NO { get; set; }
        public Nullable<short> TRANSACTION_TYPE { get; set; }
        public string CASE_ID { get; set; }
        public string PREF_LIST_ID { get; set; }
        public Nullable<int> APP_ID { get; set; }
        public string CRASH_CART_ID { get; set; }
    }
}
